### DISCUSSION TITLE
Odd behavior visits bar graph (lol... why?)

### DISCUSSION THREAD

**Discussion by c1972** [thorsten/phpMyFAQ#3033] [Q&A]
Look at that:

![besuche](https://github.com/thorsten/phpMyFAQ/assets/112912128/cbfa6349-176f-4f4f-95d4-0a7cac0f1cb5)

lol? 🤪 🤔 

Regards Christian

**Comment by thorsten** [comment]
Which browser do you use?

**Comment by c1972** [comment]
I know the new stats are way better. Just wondered... 

OS: Windows
Browser: Chrome 126.0.6478.127
phpMyFAQ Version: 3.2.7
PHP-Version: 8.2
Database: MariaDB 10.11.4
Elasticsearch: No
